sklearn/metrics/tests/test_common.py

```diff
@@ -233,7 +233,8 @@ def precision_recall_curve_padded_thresholds(*args, **kwargs):
     "ovo_roc_auc": partial(roc_auc_score, average="macro", multi_class='ovo'),
     "weighted_ovo_roc_auc": partial(roc_auc_score, average="weighted",
                                     multi_class='ovo'),
-    "partial_roc_auc": partial(roc_auc_score, max_fpr=0.5),
+    "partial_roc_auc_max_fpr": partial(roc_auc_score, max_fpr=0.5),
+    "partial_roc_auc_min_tpr": partial(roc_auc_score, min_tpr=1),
```

Comment:
Here you are setting min_tpr to  1 in the test fixture for partial_roc_auc_min_tpr. This is why it complains about the value being out of range. What's your intention here?

Reply:
> Here you are setting min_tpr to 1 in the test fixture for partial_roc_auc_min_tpr. This is why it complains about the value being out of range. What's your intention here?

Ahh, good cache, that is a typo. The `min_tpr` should be in range [0, 1). I changed it to 0.5 now and the number of test_common errors reduced from 13 to 8